src/web_interface/templates/show_statistic.html

```diff
@@ -68,6 +68,11 @@ <h3 class="mb-3">Firmware Statistics</h3>
                     {{ macros.stats_table_row("unique included files total size", stats["general_stats"]["total_file_size"] | number_format) }}
                     {{ macros.stats_table_row("unique included files average size", stats["general_stats"]["average_file_size"] | number_format) }}
                 {% endif %}
+                {% if "device_class_count" in stats["general_stats"] %}
```

Comment:
The _unique_ before each stat is unnecessary. If they were not unique, then all values would just be == firmware samples

Reply:
changed